exampleSite/content/docs/guide/shortcodes/tabs.md

```diff
@@ -91,3 +91,29 @@ Markdown syntax including code block is also supported:
   {{< /tab >}}
 
 {{< /tabs >}}
+
+
+### Use Code Blocks
```

Comment:
@saurabh-mish I've already added. 

https://imfing.github.io/hextra/docs/guide/shortcodes/tabs/#use-markdown

Can you remove this part?

Reply:
Done